packages/api/src/utils.js

```diff
@@ -174,4 +175,63 @@ const calculateInterval = (start, end) => {
   }, intervalsInRFC[0])
 }
 
-module.exports = { hash, decodeStateTransition, getKnex, checkTcpConnect, calculateInterval }
+const getLabelBuffer = (text) =>
+  Buffer.from(`${
+    Buffer.from(`12${`0${(text.length).toString(16)}`.slice(-2)
+    }`, 'hex')}${text}`
+  )
+
+const validateAliases = async (aliases, identifier, dapi) => {
+  const aliasesWithContestedState = await Promise.all(aliases.map(async (alias) => {
+    const [label, domain] = alias.split('.')
+
+    const normalizedLabel = label.toLowerCase().replace(/[oli]/g, (match) => {
```

Comment:
use standarized function to convert normalizedLabel

Reply:
✅